kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/KroxyliciousTesters.java

```diff
@@ -51,4 +64,10 @@ public static KroxyliciousTester kroxyliciousTester(ConfigurationBuilder builder
     public static MockServerKroxyliciousTester mockKafkaKroxyliciousTester(Function<String, ConfigurationBuilder> configurationForMockBootstrap) {
         return new MockServerKroxyliciousTester(MockServer.startOnRandomPort(), configurationForMockBootstrap);
     }
-}
+
+    public record TesterSetup(Function<String, ConfigurationBuilder> configurationBuilderFunction, Optional<KeytoolCertificateGenerator> certificateGenerator) {
```

Comment:
This is very definitely a crutch to get development moving. Its not intended to be the long term answer. 

I was thinking about using to also specify the type of tester to create (hence using a signature that would fit the mock tester) but stopped short of doing that to focus on the topic management aspects.

Reply:
I think I commented on a discussion already, I think we might want to refactor KeytoolCertificateGenerator to have a better public API.  IIRC we can probably lift some code from Strimzi to be a starting point.

Reply:
So I've got rid of the `TesterSetup` record and replaced it with a builder for testers. 

I'm now seeing if that builder can replace the `testers` class by making things like mock setup optional